backend/lcfs/web/api/fuel_export/services.py

```diff
@@ -203,7 +204,7 @@ def fuel_type_row_mapper(self, compliance_period, fuel_types, row):
                 default_carbon_intensity=(
                     round(row_data["default_carbon_intensity"], 2)
                     if row_data["fuel_type"] != "Other"
-                    else default_ci.get(row_data["category"])
+                    else round(category_ci, 2) if category_ci is not None else None
```

Comment:
If this needs to be rounded like the other CI, lets round it in the schema like the other ones as well.

Reply:
Passed to the schema.